gapis/api/subcmd_idx.go

```diff
@@ -82,3 +82,14 @@ func (s *SubCmdIdx) Decrement() {
 		*s = (*s)[:len(*s)-1]
 	}
 }
+
+// Contains returns true if s is one of the parent nodes of s2 or equals to s2.
+func (s SubCmdIdx) Contains(s2 SubCmdIdx) bool {
```

Comment:
Is it not slightly simpler and more efficient to implement this as:

```go
func (s SubCmdIdx) Contains(s2 SubCmdIdx) bool {
	if len(s2) < len(s) {
		return false
	}
	return s.Equals(s2[:len(s)])
}
```

or even:


```go
func (s SubCmdIdx) Contains(s2 SubCmdIdx) bool {
	return len(s2) >= len(s) && s.Equals(s2[:len(s)])
}
```